python/cuml/dask/common/comms.py

```diff
@@ -178,6 +175,26 @@ def _func_init_nccl(sessionId, uniqueId):
         print("An error occurred initializing NCCL!")
 
 
+class ListenerThread(threading.Thread):
+    def __init__(self, verbose):
+        threading.Thread.__init__(self)
+        self.listener = ucp.create_listener(_connection_func)
+        self.port = self.listener.port
+        self.verbose = verbose
+
+    def run(self):
+        if self.verbose:
+            print("Running listener thread")
+        while not self.listener.closed():
+            time.sleep(0.01)
```

Comment:
10ms seems a little short to me, since this is a thread it can presumably disrupt the main thread by interacting with the gil every 10ms, but it's not the end of the world

Reply:
Really, this doesn't have to be anything small. I'll just raise it to 1s